src/Angular.js

```diff
@@ -1913,7 +1913,7 @@ function bindJQuery() {
 
   // Use jQuery if it exists with proper functionality, otherwise default to us.
   // AngularJS 1.2+ requires jQuery 1.7+ for on()/off() support.
-  // AngularJS 1.3+ technically requires at least jQuery 2.1+ but it may work with older
+  // AngularJS 1.8+ technically requires at least jQuery 2.1+ but it may work with older
```

Comment:
This note should stay as it was, I think. We changed supported jQuery versions in 1.3 and it stayed that way.

Reply:
👍